eslint.config.js

```diff
@@ -105,7 +110,6 @@ module.exports = [
 			"test/unit/**"
 		],
 		ignores: [
-			"test/data/jquery-3.7.1.js",
```

Comment:
I'd keep it, perhaps even generalized to `test/data/jquery-*`. It's essentially a vendor file and we should not lint it, even if the lint passes - especially that future changes to ESLint or our config may make it not pass anymore.

Reply:
If you do generalize it, please backport it to `main` as well.

Reply:
It's not linted fyi. eslint 9 was shooting warnings about this file, so I moved it to global ignores at the top, which avoids parsing it altogether. That said, I forgot you had requested this be generic in the past so now is a good time for that.

Reply:
It's generalized now